build/repo.targets

```diff
@@ -106,19 +106,11 @@
       Condition="'$(CloneRepositoryCommit)'!=''" />
   </Target>
 
-  <Target Name="_CopyKorebuild" AfterTargets="_CloneRepository">
-    <ItemGroup>
-      <_CopyItems Include="$(RepositoryRoot).build\**\*.*" />
-    </ItemGroup>
-    
-    <Message Text="Copying Korebuild from Universe to repository $(_CloneRepositoryRoot)$(CloneRepository)" />
-    <Copy SourceFiles="@(_CopyItems)" DestinationFolder="$(_CloneRepositoryRoot)$(CloneRepository)\.build\%(RecursiveDir)" />
-  </Target>
-  
   <Target Name="BuildRepositories"
      DependsOnTargets="_FilterRepositories;_FindDotNetPath;_GenerateRestoreGraphSpecs;_GenerateBuildGraph;_UpdateNuGetConfig;_CreateRepositoriesListWithCommits">
 
     <PropertyGroup>
+      <_RepoDir>$(_CloneRepositoryRoot)%(Repository.Identity)</_RepoDir>
```

Comment:
I'm not sure this works. This would basically create one property with the identity of the last repository. You might need a separate target that loops over individual projects for this.